guide/menu.md

```diff
@@ -51,6 +51,10 @@
     - [MATLAB](matlab.md)
     - [Controller Plugin](controller-plugin.md)
     - [Webots Plugin](webots-plugin.md)
+- [Simulation Export](simulation-export.md)
```

Comment:
I like `Web interface`, because we will probably document soon the WWI/supervisor feature there.